src/fabric/ScreenStackHeaderConfigNativeComponent.ts

```diff
@@ -44,6 +44,7 @@ export interface NativeProps extends ViewProps {
   backButtonDisplayMode?: WithDefault<BackButtonDisplayMode, 'default'>;
   hideBackButton?: boolean;
   backButtonInCustomView?: boolean;
+  blurEffect?: string;
```

Comment:
I don't know if I'm fond of setting blurEffect as a string. In our `types.tsx`, we're using a special [type](https://github.com/software-mansion/react-native-screens/blob/main/src/types.tsx#L42) for defining every possible string that could be a blurEffect. I'm concerned if there could be a situation, where invalid `blurEffect` entered here could just bring a crash on iOS. Some day ago, there was [that](https://github.com/software-mansion/react-native-screens/pull/1751) PR that was also adding a support for `blurEffect` on Fabric. Sadly, it has been abandoned, but maybe you could try to actually enter the type to the Spec and check if it'll just work the same as with `string`?